### PR TITLE
Fix terraform-google-conversion v5 imports

### PR DESCRIPTION
**Pull request by melinath**
@iyabchen @amirkaromashkin @rileykarson FYI

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

File: mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb [hunk 1]
```diff
@@ -127,7 +127,6 @@ func TestAccContainerCluster_misc(t *testing.T) {
 }
 
 func TestAccContainerCluster_withAddons(t *testing.T) {
-	t.Skipf("Skipping test %s due to https://github.com/hashicorp/terraform-provider-google/issues/16114", t.Name())
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb:130]: drive-by: I think you want to revert the commit that created this change- you probably started from a dirty branch rather than `main`?

**Comment by melinath** [comment]
TGC integration test failures are expected due to the generated code missing the go.mod change - https://github.com/GoogleCloudPlatform/terraform-google-conversion/pull/1737 has the code changes and go.mod change and has passing integration tests.

**Review by zli82016 — APPROVED** [review]
LGTM.
